.eslintrc.js

```diff
@@ -5,53 +5,53 @@ module.exports = {
     },
     "extends": "eslint:recommended",
     "rules": {
-        "no-global-assign": 1,
+        "no-global-assign": 2,
```

Comment:
According to the [online docs](https://eslint.org/docs/user-guide/configuring#configuring-rules), this can be ``2`` or ``"error"``. Can we use the text version instead?